src/testdir/test_filetype.vim

```diff
@@ -659,7 +660,7 @@ def s:GetFilenameChecks(): dict<list<string>>
     samba: ['smb.conf'],
     sas: ['file.sas'],
     sass: ['file.sass'],
-    sather: ['file.sa'],
+    sather: ['file.sather'],
```

Comment:
In `filetype.vim`, we are only detecting `*.sa` files as potential `sather filetype`. Did you mean to add a file pattern `*.sather` as well?